main/mainworker.cpp

```diff
@@ -12988,8 +12988,8 @@ bool MainWorker::SetSetPointInt(const std::vector<std::string>& sd, const float
 
 		if (
 			(value_unit.empty())
-			|| (value_unit == "�C")
-			|| (value_unit == "�F")
+			|| (value_unit == "°C")
```

Comment:
Thank you for reverting this.
But github still thinks there is something changed otherwise it would not show the delete/addition

Reply:
I will look into this.